audit_engine/core/schemas.py

```
@@ -0,0 +1,187 @@
+    # Extended fields for agentic AI integration
+    cross_chain_impact: Optional[List[str]] = Field(None, description="Affected blockchain networks")
+    remediation_suggestion: Optional[str] = Field(None, description="AI-generated patch suggestion")
+    explainability_trace: Optional[Dict[str, Any]] = Field(None, description="Decision trace for explainability")
+    rl_feedback_score: Optional[float] = Field(None, description="Reinforcement learning feedback score")
+
+    class Config:
+        use_enum_values = True
+        json_encoders = {
+            datetime: lambda v: v.isoformat(),
+            UUID: lambda v: str(v)
+        }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)